### DISCUSSION TITLE
Information about used antenna

### DISCUSSION THREAD

**Discussion by gsawczuk** [wavelog/wavelog#182] [Ideas]
Hi,

Would it be possible to add an additional field in the form describing my station so that next to the type of transceiver used, there would be an option to select one of the predefined antennas? I mean a simple select field where we choose from globally defined antennas. E.g. "half-wave dipole 40m", "EF", "3el Yagi 10m", etc. After saving the QSO, the current selection would remain active until you manually switched the type of antenna used.

What do you think about such functionality? It seems that it should not be difficult to implement, and even the ADIF format takes into account the MY_ANTENNA field of the string type in which such information should be stored.

Vy 73 de Greg SP6GSA

**Comment by HB9HIL** [comment]
Hello @gsawczuk 

this is a fair point and we will take a look at that. We are currently working on some stuff which is related to locations and qso logging. So please stay tight and have a little bit patience :) 

we will adress that :) 

73

**Comment by ve9qrp** [comment]
May I add my support for Greg's idea and add that people doing this would also probably like to be able to record in similar fields the receiver and transmitter they used, especially if that radio was not connected by CAT control. For instance, I have a boatanchor receiver and a home-made regenerative one. It would be fun to have a record of which of these were used in certain QSOs. (Perhaps, though, the developers would recommend users employ custom hashtags in their Notes field.)

**Comment by ekiojp** [comment]
I also was looking for feature, been able to read/export MY_ANTENNA and MY_RIG from ADIF format as printing QSL with this information is very common amount JA operators
